grammar.js

```diff
@@ -202,7 +203,11 @@ module.exports = grammar({
 
   rules: {
     // Top-level rules.
-    program: $ => repeat(choice($._expression, $._semicolon, $._newline)),
+    // The zero width `$._start` ensures that `program` starts at `(0, 0)`.
+    program: $ => seq(
+        $._start,
+        repeat(choice($._expression, $._semicolon, $._newline))
+    ),
```

Comment:
Main change 1 of 2

We change `program` slightly so that it must start with the zero width `$._start` external. This also makes `START` a "valid symbol"  in this one very specific context according to `valid_symbols` at the C level.